### ISSUE TITLE
Breaks React-Dropzone after 3.0.0 update

### ISSUE THREAD

**Issue by samchouse** [aidenybai/million#1025]
### What version of `million` are you using?

3.0.6

### Are you using an SSR adapter? If so, which one?

No

### What package manager are you using?

npm

### What operating system are you using?

Linux

### What browser are you using?

Chrome, Firefox

### Describe the Bug

Million doesn't render React Dropzone properly anymore.

### What's the expected result?

It renders properly.

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/vitejs-vite-aiawis?file=src%2FApp.tsx

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by tobySolutions** [comment]
Hey there @Xenfo , are you sure this only started happening after Million v3? I have reported this to the team already though. Thank you very much.

**Comment by Xenfo** [comment]
Yeah, I was using 2.6.4 before upgrading and hadn't gotten this issue.

**Comment by Xenfo** [comment]
Not stale
